FMC increased leniency (#34)
CarterKoala/wca-regulations-wrc: wca-regulations.md

        - E2a1) Before the attempt has begun, the competitor must not write on any paper intended for the attempt. Exception: the competitor may write information to identify the attempt (see [Regulation E2c1](regulations:regulation:E2c1)). If they write this information on the paper that contains the scramble sequence, they must write only on the side without the scramble sequence before the attempt. Penalty for writing anything other than information to identify the attempt: disqualification of the attempt (DNF).
    - E2b) All competitors have a total time limit of 60 minutes to find and write a solution.
        - E2b1) The judge should call "5 MINUTES REMAINING" at 55 minutes, and must call "STOP" at 60 minutes.
    - E2c) At 60 minutes, each competitor must submit to the judge a single piece of paper with a written solution and information to identify the attempt. The competitor's solution is considered correct if starting with a solved puzzle and applying the scramble sequence followed by the solution results in a solved puzzle. Penalty for an incorrect solution: disqualification of the attempt (DNF).

>         - E2c1) Information to identify the attempt is: the competitor's name, WCA ID, or competition registrant ID (at least one, optionally multiple), and optionally the competition name, round, or attempt number. Penalty for a submitted solution without name, WCA ID, or competition registrant ID: disqualification of the attempt (DNF). Information to identify the attempt may be added by the competitor after the attempt has been submitted but before the end of the competition, at the discretion of the judge and under their direct supervision. The competitor must not write anything after the time limit without the judge's supervision and must not make any changes to the solution. Penalty: disqualification of the attempt (DNF).

Reviewer: ```suggestion
        - E2c1) Information to identify the attempt is: the competitor's name, WCA ID, or competition registrant ID (at least one, optionally multiple), and optionally the competition name, round, or attempt number. Penalty for a submitted solution without information to identify the attempt: disqualification of the attempt (DNF). Information to identify the attempt may be added by the competitor after the attempt has been submitted but before the end of the competition, at the discretion of the judge and under their direct supervision. The competitor must not write anything after the time limit without the judge's supervision and must not make any changes to the solution. Penalty: disqualification of the attempt (DNF).
```
Consistent use of defined terminology. I don't particularly like that the penalty for writing after time is up is hidden in the regulation for identifying your attempt, but this isn't new.

        - E2c2) The solution must be an unambiguous sequence of moves with each move written out in sequential order. Penalty for an ambiguous solution: disqualification of the attempt (DNF).
        - E2c3) The competitor should make it clear which moves are intended to be part of their solution, by clearly blacking out/scribbling out any moves to exclude, or by drawing a single outline around their solution.

>         - E2c4) Only numbers, letters, and apostrophes will be considered as part of a competitor’s solution and other symbols will be disregarded. The remaining solution must only use moves that are exactly defined as notation for the 3x3x3 Cube in [Regulation 12a](regulations:regulation:12a). Penalty: disqualification of the attempt (DNF).

Reviewer: ```suggestion
        - E2c4) Only numbers, letters, and apostrophes are considered as part of a competitor’s solution. Other symbols are disregarded. The remaining solution must only use moves that are exactly defined as notation for the 3x3x3 Cube in [Regulation 12a](regulations:regulation:12a). Penalty: disqualification of the attempt (DNF).
```
Multiple sentences for clarity. Switch to present tense, which is more common than future in the regs.

>         - E2c6) If a competitor incorrectly capitalizes a move that is defined in [Regulation 12a](regulations:regulation:12a), it should be interpreted as if it were correctly capitalized and accepted as valid notation.

Reviewer: ```suggestion
        - E2c6) If a competitor incorrectly capitalizes, or writes in lower case, a move that is defined in [Regulation 12a](regulations:regulation:12a), it should be interpreted as if it has the correct capitalization and accepted as valid notation.
```
Because using capitalize as a verb implies only lower to upper case, and not necessarily the reverse

Reviewer 2: ```suggestion
        - E2c6) If a competitor used incorrect capitalization for a move defined in [Regulation 12a](regulations:regulation:12a), it should be interpreted as if it has the correct capitalization and accepted as valid notation.
```

        - E2c7) Any writing or other markings on the solution sheet which are clearly not a part of the competitor's intended solution may be disregarded, at the discretion of the judge.

>         - E2c8) Competitors must not intentionally make their solution harder to judge by using external symbols, incorrect capitalization, or by any other means. Penalty: disqualification of the attempt (DNF) at the discretion of the WCA Delegate.

Reviewer: ```suggestion
        - E2c8) Competitors must not intentionally make their solution harder to judge by using moves or symbols not defined as notation for the 3x3x3 Cube, incorrect capitalization, or by any other means. Penalty: disqualification of the attempt (DNF) at the discretion of the WCA Delegate.
```
"External symbols" was not clear.

        - E2b1) The judge should call "5 MINUTES REMAINING" at 55 minutes, and must call "STOP" at 60 minutes.

>     - E2c) At 60 minutes, each competitor must submit to the judge a single piece of paper with a written solution and information to identify the attempt.

Reviewer 2: ```suggestion
    - E2c) At 60 minutes, each competitor must submit to the judge a single piece of paper with a written solution and information to identify the attempt. The competitor's solution is considered correct if starting with a solved puzzle and applying the scramble sequence followed by the solution results in a solved puzzle. Penalty for an incorrect solution: disqualification of the attempt (DNF).
```
I think this makes sense to put here. This also avoids the awkward renumbering of E2c5 (instead we can just skip it)
This would be in conjunction with the other suggestions

>         - E2c1) Information to identify the attempt is: the competitor's name, WCA ID, or competition registrant ID (at least one, optionally multiple), and optionally the competition name, round, or attempt number. Penalty for a submitted solution without name, WCA ID, or competition registrant ID: disqualification of the attempt (DNF).

Reviewer 2: ```suggestion
        - E2c1) Information to identify the attempt is: the competitor's name, WCA ID, or competition registrant ID (at least one, optionally multiple), and optionally the competition name, round, or attempt number. Penalty for a submitted solution without name, WCA ID, or competition registrant ID: disqualification of the attempt (DNF). Information to identify the attempt may be added by the competitor after the attempt has been submitted but before the end of the competition, at the discretion of the judge and under their direct supervision. The competitor must not write anything after the time limit without the judge's supervision and must not make any changes to the solution. Penalty: disqualification of the attempt (DNF).
```

        - E2c2) The solution must be an unambiguous sequence of moves with each move written out in sequential order. Penalty for an ambiguous solution: disqualification of the attempt (DNF).

>         - E2c3) The competitor should make it clear which moves are intended to be part of their solution, by clearly blacking out/scribbling out any moves to exclude, or by drawing a single outline around their solution.

Reviewer 2: I don't think the update to E2c3 is necessary here - it is just adding more words without changing much

Reviewer 3: I think explicitly defining that drawing an outline is a valid way to highlight the solution can't hurt, even though it probably belongs into the guidelines

>         - E2c4) Only numbers, letters, and apostrophes will be considered as part of a competitor’s solution and other symbols will be disregarded. The remaining solution must only use moves that are exactly defined as notation for the 3x3x3 Cube in [Regulation 12a](regulations:regulation:12a). Penalty: disqualification of the attempt (DNF).

Reviewer 2: ```suggestion
        - E2c4) The competitor's solution should only use moves defined as notation for the 3x3x3 Cube in [Regulation 12a](regulations:regulation:12a). Any writing or other markings on the solution sheet which are clearly not a part of the competitor's intended solution or do not resemble any defined notation may be disregarded. Competitors must not intentionally make their solution harder to judge with excessive use of external symbols, incorrect capitalization, or by any other means. Penalty: disqualification of the attempt (DNF), at the discretion of the WCA Delegate.
```
I believe this encompasses the current E2c4, E2c6, and E2c7+, without being overly complicated.

Reviewer 3: One admittedly quite minor concern I have about this is that it conflates symbols and moves. What would we do for example, if a competitor wrote `R^ A'`. I believe the most consistent (albeit somewhat nonsensical) interpretation would have been `R'`, but it's not entirely clear in your variant.

Reviewer 3: We could either interpret `R^` as `R`, but what would we do with the `A'`? It is clearly part of the intended solution, and the `'` is part of notation, but not in combination with the `A`.

Reviewer 2: Just added the following to the suggestion:
` or do not resemble any defined notation `

Under this, `A'` would just be disregarded. I think `R^` would likely be interpreted as `R'` depending on what exactly it looks like. The apostrophe would be disregarded as part of the `A'` - apostrophe on its own is not defined as notation, but `R'`,`U'`, etc are.

Reviewer 3: The Eleanor Sinnott suggestion defined a fairly deterministic procedure that turns a submitted solution into an algorithm that can be executed on a cube.
1. Ignore all symbols that are not UDRLFBxyz, a number or a apostrophe.
2. Optionally ignore all symbols that are clearly not supposed to be part of the solution
3. Fix the capitalization of each remaining symbol
4. Apply the moves in normal reading order.
5. If there is something invalid, the attempt is DNF

This turns for `(R [U])'` into `R U'` (valid), and `(R U2)2` into `R U22` (invalid). I'm not sure if your definition is deterministic in the same way. `(R [U])'` could either be turned into `R U` or `R U'`, depending on your interpretation.

>         - E2c5) If a competitor incorrectly capitalizes a move that is defined in [Regulation 12a](regulations:regulation:12a), it should be interpreted as if it were correctly capitalized and accepted as valid notation.
>         - E2c6) Any writing or other markings on the solution sheet which are clearly not a part of the competitor's intended solution may be disregarded, at the discretion of the judge.
>         - E2c7) The competitor's solution is considered correct if starting with a solved puzzle and applying the scramble sequence followed by the solution (after applying any considerations from [E2c4](regulations:regulation:E2c4), [E2c5](regulations:regulation:E2c5), and [E2c6](regulations:regulation:E2c6)) results in a solved puzzle. Penalty for an incorrect solution: disqualification of the attempt (DNF).

Reviewer 2: ```suggestion
        - E2c6) If a competitor incorrectly capitalizes a move that is defined in [Regulation 12a](regulations:regulation:12a), it should be interpreted as if it were correctly capitalized and accepted as valid notation.
        - E2c7) Any writing or other markings on the solution sheet which are clearly not a part of the competitor's intended solution may be disregarded, at the discretion of the judge.
        - E2c8) Competitors must not intentionally make their solution harder to judge by using external symbols, incorrect capitalization, or by any other means. Penalty: disqualification of the attempt (DNF) at the discretion of the WCA Delegate.
```
